Assets/Scripts/UI/PauseScrn/Navigation.cs

```diff
@@ -54,7 +55,7 @@ void Update()
                 ResumeSelected();
                 OptionsDeSelected();
                 NotebookDeSelected();
-                if (Input.GetKeyDown(KeyCode.Return))
+                if (Input.GetButtonDown("Interact"))
                 {
```

Comment:
Yeah your right, it doesproceed the dialogue, maybe we should deactivate the pause screen when any dialog is playing.